usaspending_api/etl/es_award_template.json

```diff
@@ -541,7 +541,7 @@
         "tas_components": {
           "type": "keyword"
         },
-        "disaster_emergency_fund_codes": {
+        "disaster_emergency_fund_code": {
```

Comment:
Also, please create a `Data Change` sub-task under DEV-6078 ... or since there may be 6 of these stories now, maybe a standalone `Data Change` ticket ... which can be referenced in the deploy coord steps for this Sprint's deploy.

Reply:
Is this change even necessary? I would investigate if the field can remain before we start talking about Data Change tickets

Reply:
No it;s not necessary, and has been changed